### PR TITLE
Magic Links: Wires new Endpoints

### PR DESCRIPTION
**Pull request by jleandroperez**
### ⚠️ Important
- [x] Make sure the new endpoints are deployed before merging
- [x] Remove the staging URL!

### Fix
In this PR we're wiring the new code-based endpoints.

Ref. #1611

### Test
1. Logout from Simplenote macOS
2. Click on the `Log In` button
3. Enter your email
4. Click on `Instantly Log In with Email`

- [x] Verify that shortly after you get the Auth Email
- [ ] Verify that clicking on Log Into Simplenote button causes the mac app to Log In

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Simplenote/Classes/SimplenoteConstants.swift [hunk 1]
```diff
@@ -45,8 +45,8 @@ class SimplenoteConstants: NSObject {
     ///
     static let resetPasswordURL     = currentEngineBaseURL.appendingPathComponent("/reset/?redirect=simplenote://launch&email=")
     static let settingsURL          = currentEngineBaseURL.appendingPathComponent("/settings")
-    static let loginRequestURL      = currentEngineBaseURL.appendingPathComponent("/account/request-login")
-    static let loginCompletionURL   = currentEngineBaseURL.appendingPathComponent("/account/complete-login")
+    static let loginRequestURL      = "https://magic-code-dot-simple-note-hrd.appspot.com/account/request-login" //googleAppEngineBaseURL.appendingPathComponent("/account/request-login")
+    static let loginCompletionURL   = "https://magic-code-dot-simple-note-hrd.appspot.com/account/complete-login" //googleAppEngineBaseURL.appendingPathComponent("/account/complete-login")
```
Comment (jleandroperez) [inline Simplenote/Classes/SimplenoteConstants.swift]: 🛑 Important!! This will be reverted before merging!!

**Review by roundhill — APPROVED** [review]
Worked! Your instructions said Mac app but I assume that was copy-pasta error :)

**Review by charliescheer — APPROVED** [review]

**Comment by jleandroperez** [comment]
Thank you both!!

**Pull request merged**